src/muse/quantities.py

```diff
@@ -47,6 +47,7 @@ def supply(
         production_method = maximum_production
 
     maxprod = production_method(technologies, capacity)
+    minprod = minimum_production(technologies, capacity)
```

Comment:
Would it make sense to rename `production_method` to `maximum_production`? It might be clearer.